OsmAnd/build.gradle

```diff
@@ -227,11 +227,15 @@ android {
 
 	buildTypes {
 		debug {
+			buildConfigField "String", "OPR_BASE_URL", "\"https://test.openplacereviews.org/\""
```

Comment:
OPR_BASE_URL - should be encoded inside app, and now should be always test, later will be changed always https://openplacereviews.org/